src/components/tasks/card/index.tsx

```diff
@@ -66,7 +69,6 @@ const Card: FC<Props> = ({ content }) => {
       <div className={classNames.cardItems}>
         <span className={classNames.cardSpecialFont}>
           Started
-          {' '}
```

Comment:
As you removed this space, it would look like this
`Started3 months ago`
Can you please add the space again?

Reply:
Added.